src/components/TableRow.tsx

```diff
@@ -0,0 +1,56 @@
+import { Link } from 'react-router-dom';
+
+import classNames from 'classnames';
+import { Person } from '../types';
+
+import { PersonLink } from './PersonLink';
+
+type Props = {
+  person: Person,
+  slug: string | undefined,
```

Comment:
```suggestion
  slug?: string,
```

can be s bit simplify